### PR TITLE
some changes

### PR DESCRIPTION
**Pull request by TeriVolta**

### CODE REVIEW

File: backend/backend/urls.py [hunk 1]
```diff
@@ -1,11 +1,13 @@
 from django.contrib import admin
 from django.urls import path, include
+#from backend import users
```
Comment (TeriVolta) [inline backend/backend/urls.py]: remove commented code

File: backend/backend/urls.py [hunk 2]
```diff
@@ -16,6 +18,7 @@ def get(self, request):
 
 
 urlpatterns = [
+    path('user/', include('users.urls')),
```
Comment (TeriVolta) [inline backend/backend/urls.py]: change path to `/api/auth/` and place it after  `path('api/auth/', include('rest_auth.urls')),`

File: backend/users/urls.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from users.views import UserRegisterView
+
+
+urlpatterns = [
+    path('register/', UserRegisterView.as_view(), name='user_register'),]
```
Comment (TeriVolta) [inline backend/users/urls.py]: change path to `users/` and name to `users_list`

File: backend/users/views.py [hunk 4]
```diff
@@ -6,6 +6,10 @@
 from rest_framework.response import Response
 from rest_framework.views import APIView
 
+from django.http import HttpResponse
+
+def test_f(request):
```
Comment (TeriVolta) [inline backend/users/views.py]: remove this test function

File: backend/users/views.py [hunk 5]
```diff
@@ -6,6 +6,10 @@
...
+    return HttpResponse('hello world')
 
 class UserRegisterView(APIView):
```
Comment (TeriVolta) [inline backend/users/views.py]: change name to `UsersListView`

File: backend/backend/urls.py (hunk 1 above)
Comment (Vesli1715) [reply backend/backend/urls.py]: Done

File: backend/backend/urls.py [hunk 6]
```diff
@@ -19,9 +20,10 @@ def get(self, request):
     path('admin/', admin.site.urls),
 
     path('api/auth/', include('rest_auth.urls')),
-    path('api/auth/register/', UserRegisterView.as_view(), name='user_register'),
+    path('api/auth/', include('users.urls')),
+    path('api/auth/register/', UsersListView.as_view(), name='user_register'),
```
Comment (TeriVolta) [inline backend/backend/urls.py]: No need in this url, you already included `users.urls`, remove this line

File: backend/users/urls.py [hunk 7]
```diff
@@ -0,0 +1,6 @@
+from django.urls import path
+from users.views import UsersListView
+
+
+urlpatterns = [
+    path('register/', UsersListView.as_view(), name='user_register'),]
```
Comment (TeriVolta) [inline backend/users/urls.py]: change `register/` to ``

File: backend/backend/urls.py [hunk 8]
```diff
@@ -19,9 +20,10 @@ def get(self, request):
     path('admin/', admin.site.urls),
 
     path('api/auth/', include('rest_auth.urls')),
-    path('api/auth/register/', UserRegisterView.as_view(), name='user_register'),
+    path('api/auth/', include('users.urls')),
```
Comment (TeriVolta) [inline backend/backend/urls.py]: change `api/auth/` to 'api/users/'

File: backend/backend/settings.py [hunk 9]
```diff
@@ -49,6 +49,8 @@
 
     'tracklist',
     'music_projects',
+
+    'users',
```
Comment (Vesli1715) [inline backend/backend/settings.py]: some review comment

**Pull request merged**